xt/lib/Pherkin/Extension/LedgerSMB.pm

```diff
@@ -214,13 +214,13 @@ sub ensure_template {
 sub create_from_template {
     my ($self, $company) = @_;
 
-    my $template = $self->template_db_name;
-    $self->ensure_nonexisting_company($company);
-    $self->super_dbh->do(qq(CREATE DATABASE "$company" TEMPLATE "$template"));
+        my $template = $self->template_db_name;
+        $self->ensure_nonexisting_company($company);
+        $self->super_dbh->do(qq(CREATE DATABASE "$company" TEMPLATE "$template"));
 
     $self->last_feature_stash->{"the company"} = $company;
     $self->last_scenario_stash->{"the company"} = $company;
-    $self->_clear_admin_dbh;
+        $self->_clear_admin_dbh;
```

Comment:
Any reason to indent some of the function (but not the rest)? Ideally, there would be no re-indenting here.